### ISSUE TITLE
bad follower token: netrc missing "password"?

### ISSUE THREAD

**Issue by mherold** [troykinsella/concourse-ansible-playbook-resource#23]
According to e.g. https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html, I gather the format of netrc should be: 

`"default login #{username} password #{password}\n"` instead  of 
https://github.com/troykinsella/concourse-ansible-playbook-resource/blob/89405307d393d6e93bf3178258ff22430fd0860a/assets/lib/git_config.rb#L17

I had errors complaining about a "bad follower token". For my use case, I overwrote the generated netrc with the following, which fixed the problem:

```
machine github.com login XXX password YYY
default login anonymous password user@site
```

**Comment by troykinsella** [comment]
Yeah I'm thinking one should just be able to supply the contents of the `netrc` file, fully. Aside from being incorrect, the above template is too restrictive to be useful.

**Comment by mherold** [comment]
Yes that might be useful. I mean, the current configuration is nice & easy and may cover the majority of people's use cases. On the other hand, in our use case we have multiple roles from different private repositories in our requirements.yml. If different credentials are required for each I think this is currently not possible without some workarounds / "manual" changes. Out-of-the-box I only see these possibilities:

(A) Using git username/password: Only one username/password combination can be configured (=> `netrc`)

(B) Using git ssh keys: Only one key can be configured (but: e.g. if you want to use github deploy keys and have multiple repos, you would need to configure multiple keys).
